tests/test_articles_extraction.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+from itemadapter import ItemAdapter
+
+from tests.utils_extraction import get_web_poet_response
+from zyte_test_websites.articles.extraction import (
+    TestArticleNavigationPage,
+    TestArticlePage,
+)
+
+
+async def test_article_extraction(articles_client):
+    response = await get_web_poet_response(articles_client, "/article/119")
+    page = TestArticlePage(response)
+    body = (
+        "The government of Nigeria is hoping to triple cocoa production over"
+        " the next three years with the launch of an ambitious development"
+        " programme.  Agriculture Minister Adamu Bello said the scheme aimed"
+        " to boost production from an expected 180,000 tonnes this year to"
+        " 600,000 tonnes by 2008. The government will pump 154m naira ($1.1m;"
+        " £591,000) into subsidies for farming chemicals and seedlings."
+        " Nigeria is currently the world's fourth-largest cocoa producer."
+        "  Cocoa was the main export product in Nigeria during the 1960s. But"
+        " with the coming of oil, the government began to pay less attention"
+        " to the cocoa sector and production began to fall from a peak of"
+        " about 400,000 tonnes a year in 1970. At the launch of the programme"
+        " in the south-western city of Ibadan, Mr Bello explained that an"
+        " additional aim of the project is to encourage the processing of"
+        " cocoa in the country and lift local consumption. He also announced"
+        " that 91m naira of the funding available had been earmarked for"
+        " establishing cocoa plant nurseries. The country could be looking to"
+        " emulate rival Ghana, which produced a bumper crop last year."
+        ' However, some farmers are sceptical about the proposals. "People'
+        ' who are not farming will hijack the subsidy," said Joshua Osagie,'
+        ' a cocoa farmer from Edo state told Reuters. "The farmers in the'
+        ' village never see any assistance," he added.  At the same time as'
+        " Nigeria announced its new initiative, Ghana - the world's second"
+        " largest cocoa exporter - announced revenues from the industry had"
+        " broken new records. The country saw more than $1.2bn-worth of the"
+        " beans exported during 2003-04. Analysts said high tech-production"
+        " techniques and crop spraying introduced by the government led to"
+        " the huge crop, pushing production closer to levels seen in the 1960s"
+        " when the country was the world's leading cocoa grower."
+    )
+    item = await page.to_item()
+    assert ItemAdapter(item).asdict() == {
+        "articleBody": body,
+        "articleBodyHtml": f"""<article class="article-content">
+                    <div class="text-justify">{body}</div>
+                </article>""",
```

Comment:
This doesn't look like a simplified HTML - is the output correct? I think we should use clear-html library to extract this field.